src/components/Toast/Toast.tsx

```diff
@@ -61,7 +61,8 @@ export default React.memo(function Toast(props: Props) {
         }
       };
     },
-    [appBridge, frame, props],
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [appBridge, props],
```

Comment:
We're smarter than eslint in this case. `frame` won't change so this will save us a few renders

Reply:
I was able to remove the comment because we're using deep compare now.